addons/escoria-core/game/core-scripts/esc/commands/inventory_remove.gd

```diff
@@ -1,6 +1,10 @@
-# `inventory_remove item`
+# `inventory_add item`
```

Comment:
I think there is a mistake here. The command is `inventory_remove`

Reply:
Was fixed by Duncan.